src/csharp/Grpc.Core.Tests/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "buildOptions": {
+    "compile": "**/*.cs",
+    "copyToOutput": {
+      "mappings": {
+        "nativelibs/windows_x64/grpc_csharp_ext.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
+        "nativelibs/windows_x86/grpc_csharp_ext.dll": "../../../vsprojects/Debug/grpc_csharp_ext.dll"
+      }
+    },
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Grpc.Core": {
+      "version": "0.0.1",
+      "taget": "project"
```

Comment:
Nice catch, I was wondering how come the "target": "project" doesn't work as expected. :-)